unlock-app/src/components/interface/locks/metadata/TicketForm.tsx

```diff
@@ -23,6 +25,14 @@ export function TicketForm({ disabled, lockAddress, network }: Props) {
     control,
   })
 
+  const eventPageUrl = new URL(
+    `${window.location.origin}/event?lockAddress=${lockAddress}&network=${network}`
+  )
+
+  const mapAddress = `https://www.google.com/maps/embed/v1/place?q=${encodeURIComponent(
+    ticket?.event_address || 'Ethereum'
+  )}&key=AIzaSyD_mt4bpelB7Dlr7XzfUW9k7b9agbf_iSo`
```

Comment:
Move the API key to config and use NEXT_PUBLIC_MAP_KEY so it is configurable otherwise fallback